Mathlib/Analysis/SpecialFunctions/Log/Base.lean

```diff
@@ -407,6 +447,16 @@ theorem logb_prod {α : Type*} (s : Finset α) (f : α → ℝ) (hf : ∀ x ∈
     simp only [Finset.mem_insert, forall_eq_or_imp] at hf
     simp [ha, ih hf.2, logb_mul hf.1 (Finset.prod_ne_zero_iff.2 hf.2)]
 
+protected theorem _root_.Finsupp.logb_prod {α β : Type*} [Zero β] (f : α →₀ β) (g : α → β → ℝ)
+    (hg : ∀ a, g a (f a) = 0 → f a = 0) : logb b (f.prod g) = f.sum fun a c ↦ logb b (g a c) :=
+  logb_prod _ _ fun _x hx h₀ ↦ Finsupp.mem_support_iff.1 hx <| hg _ h₀
+
+theorem logb_nat_eq_sum_factorization (n : ℕ) :
+    logb b n = n.factorization.sum fun p t => t * logb b p := by
+  simp only [logb, mul_div_assoc', log_nat_eq_sum_factorization n, Finsupp.sum, Finset.sum_div]
+
+-- TODO add other limits and continuous API lemmas analogous to those in Log.lean
```

Comment:
I didn't check. What's still missing?

Reply:
Could you clarify your question? The remaining lemmas can be found in the referenced file; I have only changed this comment to follow the file rename. Would you like me to list all of the missing lemmas?

Reply:
No, sorry, I shouldn't have been lazy. My apologies.